doc/nrf/drivers/sensor_stub.rst

```diff
@@ -1,21 +1,21 @@
 .. _sensor_stub:
 
-Simple simulated sensor driver
-##############################
+Sensor stub driver
+###################
 
 .. contents::
    :local:
    :depth: 2
 
-The sensor stub is just blind interface between sensor API and simple callback functions to fetch and get data.
+The sensor stub, a simple simulated sensor driver, is a blind interface between the sensor API and simple callback functions to fetch and get data.
```

Comment:
What is "blind interface"?

Reply:
Maybe it is not the sentence that should be used? Maybe rather "dummy interface" or "pass-through interface"? I should be an interface that does nothing with the data. Just forwards it.